src/HDInsight/HDInsight/ManagementCommands/DisableAzureHDInsightMonitoringCommand.cs

```diff
@@ -19,19 +19,17 @@
 
 namespace Microsoft.Azure.Commands.HDInsight
 {
-    [Cmdlet("Disable", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "HDInsightOperationsManagementSuite",SupportsShouldProcess = true)]
-    [Alias("Disable-" + ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "HDInsightOMS")]
-    [OutputType(typeof(OperationResource))]
-    public class DisableAzureHDInsightOMSCommand : HDInsightCmdletBase
+    [Cmdlet("Disable", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "HDInsightMonitoring",SupportsShouldProcess = true)]
```

Comment:
> @aim-for-better just for clarity, is the "Monitoring" property replacing the "OMS" property on a cluster? Or is this just a coincidental GitHub diff that says it's replacing it? 😁

We replaced Get/Enable/Disable-AzHDInsightOMS with  Get/Enable/Disable-AzHDInsightMonitoring and did some necessary changes because the sdk powershell based has changed from hyak to swagger.